mypyc/irbuild/specialize.py

```diff
@@ -425,7 +450,9 @@ def can_optimize_format(format_str: str) -> bool:
 
 
 def split_braces(format_str: str) -> List[str]:
-    # This function can only be called after format_str pass can_optimize_format()
+    """This function can only be called after format_str passes
+    'can_optimize_format()'.
+    """
```

Comment:
This should _not_ be a docstring, it doesn't explain what the function does.

Reply:
Thanks for pointing out!